quizcon/templates/main/lti_assignment.html

```diff
@@ -140,7 +140,7 @@ <h1 class="mt-2">
                                 <div class="choicebox float-end">
                                     {% for x in num_markers %}
                                     <div class="form-check answer{{x}}">
-                                        <input class="form-check-input" type="radio" name="radiodefault" value="{{x}}">
+                                        <input class="form-check-input" type="radio" name="radiodefault" value="{{x}}" aria-label="radio button input for value {{x}}">
```

Comment:
This raises an interesting accessibility question. Ideally, we'd want the aria-labels to be the answers for A, B & C, then an indication of the proximity to vertex for the in-between answers. This would be a great discussion to have with Zarina.